R/taxonomy.R

```diff
@@ -208,7 +218,19 @@ setMethod("checkTaxonomy", signature = c(x = "SummarizedExperiment"),
         ans
     }
 )
-
+#' @rdname taxonomy-methods
+#'@export
+# Function to set taxonomy ranks
+setTaxonomyRanks <- function(rank) {
+    rank <- tolower(rank)
```

Comment:
Add here input test. rank should be a character vector. Length >= 1.  Check other function for help. I think .is_a_character is a correct function for checking